assets/pages/bundles-landing/components/WaysOfSupport.jsx

```diff
@@ -5,28 +5,28 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
+import { getMemLink } from '../helpers/externalLinks';
 import WayOfSupport from './WayOfSupport';
 
+import type { MemProduct } from '../helpers/externalLinks';
+
 
 // ----- Copy ----- //
 
-const generateOnClick = (baseURL: string, intcmp: string): () => void => {
-  const params = new URLSearchParams();
-  params.append('INTCMP', intcmp);
-  const ctaLink = `${baseURL}?${params.toString()}`;
+function generateOnClick(product: MemProduct, intCmp: ?string): () => void {
 
   return () => {
```

Comment:
Since it is a one-line function, probably you can simplify this with an arrow function for the entire function.

Reply:
I did that to avoid this monstrosity:

```js
const generateOnClick = (product: MemProduct, intCmp: ?string): () => () => void => () => window.location = getMemLink(product, intCmp);

```